src/test/java/org/mockitousage/debugging/VerificationListenerCallBackTest.java

```diff
@@ -12,30 +11,22 @@
 import org.mockito.verification.VerificationEvent;
 import org.mockito.verification.VerificationMode;
 
-
 import java.lang.reflect.Method;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.Assert.assertNull;
 import static org.junit.Assert.fail;
 import static org.mockito.Mockito.*;
 
 public class VerificationListenerCallBackTest {
-    private Method invocationWanted;
-    private RememberingListener listener;
-    private MockitoFramework mockitoFramework;
-
-    @Before
-    public void setUp() throws NoSuchMethodException {
-        listener = new RememberingListener();
-        mockitoFramework = Mockito.framework();
-        mockitoFramework.addListener(listener);
-        invocationWanted = Foo.class.getDeclaredMethod("doSomething", String.class);
-    }
 
     @Test
-    public void should_call_single_listener_on_verify() {
+    public void should_call_single_listener_on_verify() throws NoSuchMethodException {
         //given
+        RememberingListener listener = new RememberingListener();
+        MockitoFramework mockitoFramework = Mockito.framework();
+        mockitoFramework.addListener(listener);
+
+        Method invocationWanted = Foo.class.getDeclaredMethod("doSomething", String.class);
```

Comment:
I suggest to inline variable `invocationWanted`, at the moment this variable is declared in the given section but has no impact on the scenario.

This apply for other tests.